dev/Scroller/Scroller.cpp

```diff
@@ -808,7 +808,9 @@ winrt::Size Scroller::ArrangeOverride(winrt::Size const& finalSize)
         viewport.Width          /*viewportWidth*/,
         viewport.Height         /*viewportHeight*/);
 
-#ifndef USE_EFFECTIVE_VIEWPORT_AND_ANCHORING_FROM_PLATFORM
+#ifdef USE_EFFECTIVE_VIEWPORT_AND_ANCHORING_FROM_PLATFORM
+    m_isAnchorElementDirty = true;
```

Comment:
This can be outside the ifdefs

Reply:
I'm leaving it as is because ClearAnchorCandidates() sets the flag too.